### PR TITLE
Attribute values pagination in product details

### PR DESCRIPTION
**Pull request by orzechdev**
I want to merge this change because... it adds pagination to attribute values selection card in product details pages.

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [x] All visible strings are translated with proper context including data-formatting
3. [x] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [x] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://saleor-3071-paginate-attributes.api.saleor.rocks/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-3109 Use pagination on attribute values](https://app.clickup.com/t/2549495/SALEOR-3109)

**Comment by patrys** [comment]
Task linked: [SALEOR-3346 Add pagination to attribute values selection card in product details pages](https://app.clickup.com/t/2549495/SALEOR-3346)

**Review by dominik-zeglen — APPROVED** [review]

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx [hunk 1]
```diff
@@ -164,6 +165,9 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
                   onReferencesRemove={handlers.selectAttributeReference}
                   onReferencesAddClick={onAssignReferencesClick}
                   onReferencesReorder={handlers.reorderAttributeValue}
+                  fetchAttributeValues={() => undefined}
+                  fetchMoreAttributeValues={undefined}
+                  onAttributeSelect={() => undefined}
```
Comment (jwm0) [inline src/pages/components/PageDetailsPage/PageDetailsPage.tsx:170]: If these have to be passed as empty functions it means that they should be optional

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx [hunk 2]
```diff
@@ -155,6 +155,7 @@ const PageDetailsPage: React.FC<PageDetailsPageProps> = ({
               {data.attributes.length > 0 && (
                 <Attributes
                   attributes={data.attributes}
+                  attributeValues={[]}
```
Comment (jwm0) [inline src/pages/components/PageDetailsPage/PageDetailsPage.tsx:158]: Should this be optional with default value of `[]` instead

**Review by jwm0 — CHANGES_REQUESTED** [review]

**Review by eCzerniak — COMMENTED** [review]
1. It is not possible to create simple product. 
2. If `Value Required` is set then an error is shown during product creation

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx (hunk 1 above)
Comment (orzechdev) [reply src/pages/components/PageDetailsPage/PageDetailsPage.tsx:170]: It shouldn't be optional, I'm going to add implementation for them in another PR-s

File: src/pages/components/PageDetailsPage/PageDetailsPage.tsx (hunk 2 above)
Comment (orzechdev) [reply src/pages/components/PageDetailsPage/PageDetailsPage.tsx:158]: same here

**Review by jwm0 — APPROVED** [review]
Code looks good

**Review by eCzerniak — APPROVED** [review]

**Pull request merged**